source/guides/administrator.rst

```diff
@@ -190,6 +189,7 @@ Learn how to support growth within Mattermost.
    
    /deployment/scaling*
    /deployment/cluster*
+   /deployment/cluster-310*
```

Comment:
This one we don't actually want to add to TOC because it's an old guide. It's linked to in https://docs.mattermost.com/deployment/cluster.html